content/research.md

```diff
@@ -86,10 +86,25 @@ ourwork:
 
 import { ResearchTopicMap, ProjectsList } from './components/custom/Research';
 
-We are a **research collective** exploring paths to [awakening societies](/awakening-society) 🌻. We produce reports, papers, methods and tools. We [welcome participation](/research-community).
+## About
+
+We are a **research collective** exploring paths to [awakening societies](/awakening-society) 🌻. We produce reports, papers, methods and tools. We welcome participation and [more info on our research community can be found here.](/research-community)
```

Comment:
Do the awakening society and research community links go somewhere? When I view the file in the GitHub repo and click on the links they don't take me to another page, but maybe they will on the live version of the page?